alerts/ldap_password_spray.py

```diff
@@ -13,8 +13,12 @@
 
 class AlertLdapPasswordSpray(AlertTask):
     def main(self):
-        self.parse_config('ldap_password_spray.conf', ['threshold_count', 'search_depth_min'])
+        self.parse_config('ldap_password_spray.conf', ['threshold_count', 'search_depth_min', 'host_exclusions'])
         search_query = SearchQuery(minutes=int(self.config.search_depth_min))
+
+        for host_exclusion in self.config.host_exclusions.split(","):
```

Comment:
Add an example test case that verifies this exclusion is working as expected?

Reply:
Added